pkg/cmd/scan/output/console.go

```diff
@@ -168,7 +168,7 @@ func (c Console) writeSummary(analysis *analyser.Analysis) {
 func prettify(resource interface{}) string {
 	res := reflect.ValueOf(resource)
 	if resource == nil || res.Kind() == reflect.Ptr && res.IsNil() {
-		return "<nil>"
+		return "<null>"
```

Comment:
Why did you change the wording here ?

Reply:
I'll revert this change as it's not related to this PR (this function was duplicated). I'll create another PR to rewrite it to null.